plugins/engine-configuration/impl/src/main/java/org/pentaho/di/engine/configuration/impl/spark/SparkRunConfigurationProvider.java

```diff
@@ -3,7 +3,7 @@
  *
  *  Pentaho Data Integration
  *
- *  Copyright (C) 2002-2017 by Hitachi Vantara : http://www.pentaho.com
+ *  Copyright (C) 2002-2019 by Hitachi Vantara : http://www.pentaho.com
```

Comment:
I think this class does not need to be part of the PR because only header was changed.